OBAKit/Mapping/MapRegionManager.swift

```diff
@@ -566,9 +580,15 @@ public class MapRegionManager: NSObject,
 
 extension MapRegionManager: LocationServiceDelegate {
     public func locationService(_ service: LocationService, authorizationStatusChanged status: CLAuthorizationStatus) {
+        // "reset" this property to change the user location annotation view as needed.
+        mapView.showsUserLocation = false
         mapView.showsUserLocation = service.isLocationUseAuthorized
     }
 
+    public func locationService(_ service: LocationService, locationChanged location: CLLocation) {
+
```

Comment:
Can this be removed? If not, add a comment like `// nop` to it to indicate that it is intentionally blank.